inc/comments-template.php

```diff
@@ -9,11 +9,14 @@
  */
 
 $talk_url = get_option( 'coral_talk_base_url' );
+$static_url = get_option( 'coral_talk_static_url', $talk_url );
 $talk_container_classes = get_option( 'coral_talk_container_classes' );
 $div_id = 'coral_talk_' . absint( rand() );
 if ( ! empty( $talk_url ) ) : ?>
 	<div class="<?php echo esc_attr( $talk_container_classes ); ?>" id="<?php echo esc_attr( $div_id ); ?>"></div>
-	<script src="<?php echo esc_url( $talk_url . '/embed.js' ); ?>" async onload="
+	<script src="<?php echo esc_url( $static_url . '/static/embed.js' ); ?>" async onload="
+		Coral.static_url = '<?php echo esc_url( $static_url ); ?>';
+		Coral.talk_url = '<?php echo esc_url( $talk_url ); ?>';
```

Comment:
As we don't consume the variables outside the render call, I wouldn't recommend attaching the variables again to the `Coral` global.

Reply:
This was a convenience reference for the client JS on the WP site, but we can refactor that elsewhere.